index.js

```diff
@@ -0,0 +1,60 @@
+const inputAuthor = document.querySelector('.input-books-author');
+const inputTitle = document.querySelector('.input-books-title');
+const bookContainer = document.querySelector('.awesome-books');
+const addBook = document.querySelector('.add-books');
+
+// empty library array for storing book as object
+const library = JSON.parse(localStorage.getItem('books') || '[]');
+
+// existing entries
+const existingEntries = JSON.parse(localStorage.getItem('books') || '[]');
+
+// Update User Interface dynamically
+
+const updateUI = () => {
+  bookContainer.innerHTML = '';
+  library.forEach((data, i) => {
+    const bookSelf = document.createElement('div');
+    bookSelf.classList.add('bookItem');
+    const newBookTitle = document.createElement('p');
+    newBookTitle.textContent = `${data.title}`;
+    const newBookAuthor = document.createElement('p');
+    newBookAuthor.textContent = `${data.author}`;
+    const button = document.createElement('button');
+    button.textContent = 'Remove';
+    const separator = document.createElement('hr');
+
+    // remove books from library
+    const removeBook = function () {
+      library.splice(this, 1);
```

Comment:
[OPTIONAL] Please consider using methods that don't mutate the array where you're storing your books.

<img width="844" alt="Screen Shot 2022-06-27 at 16 16 52" src="https://user-images.githubusercontent.com/82426682/176037066-4947129a-8911-4c1b-aa1b-02b6926b5c0a.png">